pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/extension/PythonDetailsTestDouble.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.extension;
+
+import org.gradle.api.Project;
+
+import java.io.File;
+
+public class PythonDetailsTestDouble extends PythonDetails {
+
+    private PythonVersion version = new PythonVersion("3.4");
```

Comment:
I know this is just a test, but since we never used 3.4, can we please use something higher, 3.6 or 3.5 at least, even as an example in the test.